tests/libraries/test_reentrancy_guard.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+from ethereum.tools import tester
+from ethereum.tools.tester import TransactionFailed
+from pytest import fixture, mark, lazy_fixture, raises
+
+
+@fixture(scope='session')
+def testerSnapshot(sessionFixture):
+    sessionFixture.resetSnapshot()
+    sessionFixture.uploadAndAddToController('solidity_test_helpers/ReentrancyGuardHelper.sol')
+    ReentrancyGuardHelper = sessionFixture.contracts['ReentrancyGuardHelper']
+    return sessionFixture.chain.snapshot()
+
+
+@fixture
+def testerContractsFixture(sessionFixture, testerSnapshot):
+    sessionFixture.chain.revert(testerSnapshot)
+    return sessionFixture
+
+
+def test_nonReentrant(testerContractsFixture): 
+    ReentrancyGuardHelper = testerContractsFixture.contracts['ReentrancyGuardHelper']
+    # Whitelist the contract
+    testerContractsFixture.controller.addToWhitelist(ReentrancyGuardHelper.address)
```

Comment:
This is unnecessary for this test as the reentrancy guard doesn't depend on being whitelisted.  Recommend removing it so the test is more clear as what is required for it to pass.

Reply:
I'll remove it.